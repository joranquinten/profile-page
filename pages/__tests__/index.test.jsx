import { shallow } from 'enzyme';

import Index from '../index';

describe('<Index />', () => {
  it('should render without crashing', () => {
    expect(shallow(<Index />)).toMatchSnapshot();
  });
});
