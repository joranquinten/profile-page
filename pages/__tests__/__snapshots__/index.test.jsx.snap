// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> should render without crashing 1`] = `
<pages__StyledLayout>
  <Adapter
    className="ant-layout-override"
  >
    <pages__StyledBackTop>
      <BackTop
        visibilityHeight={400}
      >
        <Icon
          theme="twoTone"
          type="rocket"
        />
      </BackTop>
    </pages__StyledBackTop>
    <Adapter>
      <About />
      <Resume />
    </Adapter>
    <Adapter>
      <VCard />
    </Adapter>
  </Adapter>
</pages__StyledLayout>
`;
